test/hca_metadata_api/test.py

```diff
@@ -791,6 +794,21 @@ def test_invalid_schema_domain(self):
         )
         self.assertEqual(expected, cm.exception.args[0].args)
 
+    def test_schema_validation(self):
+        validator = SchemaValidator()
+        test_data = {
+            'describedBy': 'https://schema.humancellatlas.org/type/file/9.6.0/sequence_file',
+            'schema_type': 'file',
+            'file_core': {'file_name': 'foo.fastq.gz', 'format': 'fastq.gz'},
+            'read_index': 'read1',
+        }
+        validator.validate_json(test_data, 'foo.json')
+
+        test_data['read_index'] = 'bad-value'
+        with self.assertRaises(AssertionError) as cm:
+            validator.validate_json(test_data, 'foo.json')
+        self.assertRegex(cm.exception.args[0].args[1], "'bad-value' is not one of")
```

Comment:
```suggestion
        self.assertRegex(cm.exception.args[0].args[1], r"^'bad-value' is not one of")
```

Otherwise we would just use `assertIn`.